AzureMonitorAgent/agent.py

```diff
@@ -204,14 +208,75 @@ def install():
                                          retry_check = retry_if_dpkg_locked,
                                          final_check = final_check_if_dpkg_locked)
 
-    default_configs = {
-        "ENABLE_MCS" : "true",
-        "MCS_ENDPOINT" : "eastus.amcs.ppe.control.monitor.azure.com",
+    default_configs = {        
+        "MCS_ENDPOINT" : "amcs.control.monitor.azure.com",
         "AZURE_ENDPOINT" : "https://management.azure.com/",
-        "ADD_REGION_TO_MCS_ENDPOINT" : "false",
-        "OMS_TLD" : "int2.microsoftatlanta-int.com",
-        "customResourceId" : "/subscriptions/42e7aed6-f510-46a2-8597-a5fe2e15478b/resourcegroups/amcs-test/providers/Microsoft.OperationalInsights/workspaces/amcs-pretend-linuxVM"
+        "ADD_REGION_TO_MCS_ENDPOINT" : "true",
+        #"OMS_TLD" : "int2.microsoftatlanta-int.com",
+        #"customResourceId" : "/subscriptions/42e7aed6-f510-46a2-8597-a5fe2e15478b/resourcegroups/amcs-test/providers/Microsoft.OperationalInsights/workspaces/amcs-pretend-linuxVM",
+        #"ENABLE_MCS" : "true",
     }
+
+    # decide the mode
+    if protected_settings is None:
+        default_configs["ENABLE_MCS"] = "true"
```

Comment:
false?

Reply:
no, if no protected settings are passed, the default mode is 3rd party mode with AMCS